php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1860,15 +1880,16 @@ class NDB_BVL_Instrument extends NDB_Page
      * Wrapper to create a field that only accepts a number, with an
      * accompanying status field.
      *
-     * @param string  $field   The database field in which the response
-     *                         will be stored
-     * @param string  $label   The question text to display
-     * @param unknown $options Does not appear to be used?
+     * @param string $field The database field in which the response
+     *                      will be stored
+     * @param string $label The question text to display
      *
      * @return void
      */
-    function addNumericElement($field, $label, $options = null)
-    {
+    function addNumericElement(
+        string $field,
+        string $label
+    ): void {
```

Comment:
@driusan @PapillonMcGill re: your comments below, this function never uses the `$options` variable so I removed it from the function signature. Similarly I removed the calling code below because the values don't actually get used.